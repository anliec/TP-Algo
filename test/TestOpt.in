OPT 1 20 22 0

OPT 1 20 22 2 2 3

ADD 2 2015 6 23 20 0 1 J
ADD 2 2015 6 23 20 1 1 J
ADD 2 2015 6 23 20 2 1 J
ADD 2 2015 6 23 20 3 1 J
ADD 2 2015 6 23 20 4 1 J
ADD 2 2015 6 23 20 5 1 J
ADD 2 2015 6 23 20 6 1 J
ADD 2 2015 6 23 20 7 1 J
ADD 2 2015 6 23 20 8 1 J
ADD 2 2015 6 23 20 9 1 V
ADD 2 2015 6 23 20 10 1 J
ADD 2 2015 6 23 20 11 1 J
ADD 2 2015 6 23 20 12 1 J
ADD 2 2015 6 23 20 13 1 J
ADD 2 2015 6 23 20 14 1 J
ADD 2 2015 6 23 20 15 1 J
ADD 2 2015 6 23 20 16 1 J
ADD 2 2015 6 23 20 17 1 J
ADD 2 2015 6 23 20 18 1 J
ADD 2 2015 6 23 20 19 1 J
ADD 2 2015 6 23 20 20 1 J
ADD 2 2015 6 23 20 21 1 J
ADD 2 2015 6 23 20 22 1 J
ADD 2 2015 6 23 20 23 1 J
ADD 2 2015 6 23 20 24 1 J
ADD 2 2015 6 23 20 25 1 J
ADD 2 2015 6 23 20 26 1 J
ADD 2 2015 6 23 20 27 1 J
ADD 2 2015 6 23 20 28 1 J
ADD 2 2015 6 23 20 29 1 J
ADD 2 2015 6 23 20 30 1 J
ADD 2 2015 6 23 20 31 1 J
ADD 2 2015 6 23 20 32 1 J
ADD 2 2015 6 23 20 33 1 J
ADD 2 2015 6 23 20 34 1 J
ADD 2 2015 6 23 20 35 1 J
ADD 2 2015 6 23 20 36 1 J
ADD 2 2015 6 23 20 37 1 J
ADD 2 2015 6 23 20 38 1 J
ADD 2 2015 6 23 20 39 1 J
ADD 2 2015 6 23 20 40 1 J
ADD 2 2015 6 23 20 41 1 J
ADD 2 2015 6 23 20 42 1 J
ADD 2 2015 6 23 20 43 1 J
ADD 2 2015 6 23 20 44 1 J
ADD 2 2015 6 23 20 45 1 J
ADD 2 2015 6 23 20 46 1 J
ADD 2 2015 6 23 20 47 1 J
ADD 2 2015 6 23 20 48 1 J
ADD 2 2015 6 23 20 49 1 J
ADD 2 2015 6 23 20 50 1 J
ADD 2 2015 6 23 20 51 1 J
ADD 2 2015 6 23 20 52 1 J
ADD 2 2015 6 23 20 53 1 J
ADD 2 2015 6 23 20 54 1 J
ADD 2 2015 6 23 20 55 1 J
ADD 2 2015 6 23 20 56 1 J
ADD 2 2015 6 23 20 57 1 J
ADD 2 2015 6 23 20 58 1 J
ADD 2 2015 6 23 20 59 1 J
ADD 2 2015 6 23 21 0 1 J
OPT 1 20 21 1 2

ADD 3 2015 6 23 20 0 1 J
ADD 3 2015 6 23 20 1 1 J
ADD 3 2015 6 23 20 2 1 J
ADD 3 2015 6 23 20 3 1 V
ADD 3 2015 6 23 20 4 1 J
ADD 3 2015 6 23 20 5 1 J
ADD 3 2015 6 23 20 6 1 J
ADD 3 2015 6 23 20 7 1 J
ADD 3 2015 6 23 20 8 1 J
ADD 3 2015 6 23 20 9 1 J
ADD 3 2015 6 23 20 10 1 J
ADD 3 2015 6 23 20 11 1 J
ADD 3 2015 6 23 20 12 1 J
ADD 3 2015 6 23 20 13 1 J
ADD 3 2015 6 23 20 14 1 J
ADD 3 2015 6 23 20 15 1 J
ADD 3 2015 6 23 20 16 1 J
ADD 3 2015 6 23 20 17 1 J
ADD 3 2015 6 23 20 18 1 J
ADD 3 2015 6 23 20 19 1 J
ADD 3 2015 6 23 20 20 1 J
ADD 3 2015 6 23 20 21 1 J
ADD 3 2015 6 23 20 22 1 J
ADD 3 2015 6 23 20 23 1 J
ADD 3 2015 6 23 20 24 1 J
ADD 3 2015 6 23 20 25 1 J
ADD 3 2015 6 23 20 26 1 J
ADD 3 2015 6 23 20 27 1 J
ADD 3 2015 6 23 20 28 1 J
ADD 3 2015 6 23 20 29 1 J
ADD 3 2015 6 23 20 30 1 J
ADD 3 2015 6 23 20 31 1 J
ADD 3 2015 6 23 20 32 1 J
ADD 3 2015 6 23 20 33 1 J
ADD 3 2015 6 23 20 34 1 J
ADD 3 2015 6 23 20 35 1 J
ADD 3 2015 6 23 20 36 1 J
ADD 3 2015 6 23 20 37 1 J
ADD 3 2015 6 23 20 38 1 J
ADD 3 2015 6 23 20 39 1 J
ADD 3 2015 6 23 20 40 1 J
ADD 3 2015 6 23 20 41 1 J
ADD 3 2015 6 23 20 42 1 J
ADD 3 2015 6 23 20 43 1 J
ADD 3 2015 6 23 20 44 1 J
ADD 3 2015 6 23 20 45 1 J
ADD 3 2015 6 23 20 46 1 J
ADD 3 2015 6 23 20 47 1 J
ADD 3 2015 6 23 20 48 1 J
ADD 3 2015 6 23 20 49 1 J
ADD 3 2015 6 23 20 50 1 J
ADD 3 2015 6 23 20 51 1 J
ADD 3 2015 6 23 20 52 1 J
ADD 3 2015 6 23 20 53 1 J
ADD 3 2015 6 23 20 54 1 J
ADD 3 2015 6 23 20 55 1 J
ADD 3 2015 6 23 20 56 1 J
ADD 3 2015 6 23 20 57 1 J
ADD 3 2015 6 23 20 58 1 J
ADD 3 2015 6 23 20 59 1 J
ADD 3 2015 6 23 21 0 1 J
OPT 1 20 21 2 2 3
EXIT
