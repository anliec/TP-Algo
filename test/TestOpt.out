1 20 0 0
1 20 0 20
1 20 9 1
1 20 1 3
